Topics/Development_Process/Software_Development_Life_Cycle.md

```diff
@@ -0,0 +1,44 @@
+# Introduction to the Software Development Life Cycle (SDLC)
+### What is SDLC?
+
+The software development lifecycle (SDLC) is the cost-effective and time-efficient process that development teams use to design and build high-quality software. The purpose of this process is to provide a structured manner in which we can develop software in order to minimize project risks through forward planning. The stages of this process are as follows:
```

Comment:
I think when you said "The stages of this process are as follows:" I was expecting the stages next. Maybe you could remove this line since it comes up later?